packages/ceramic-core/src/dispatcher.ts

```diff
@@ -4,6 +4,7 @@ import CID from 'cids'
 import cloneDeep from 'lodash.clonedeep'
 
 import type Document from "./document"
+import { IpfsUtils } from "@ceramicnetwork/ceramic-common/lib"
```

Comment:
Any reason for importing from `/lib` here?

Reply:
Thanks for noticing! Webstorm does that automatically